### DISCUSSION TITLE
serversettings.json

### DISCUSSION THREAD

**Discussion by Rikkuzen** [jagrosh/MusicBot#1457] [Troubleshooting]
[14:06:40] [WARN] [Settings]: Failed to load server settings (this is normal if no settings have been set yet): java.nio.file.NoSuchFileException: serversettings.json

Getting this warning. I tried everything I could find on the net and on the issues/troubleshooting topics. 

I'm new to this so bare with me. Jagrosh mentioned something about setting server settings? Where do I do that? 

No problem with the bot when I try the rar launch and java launch thing but when I try to set it up using nssm I can't make it online.

**Comment by jagrosh** [comment]
`this is normal if no settings have been set yet`

You can ignore that warning if you haven't set sever-specific settings.
